statics/js/components/graph-layout.js

```diff
@@ -915,6 +916,34 @@ TopologyGraphLayout.prototype = {
     this.update();
   },
 
+  collapseByLevel: function(level, collapse, groups) {
+    if (level == 0) {
+      var i;
+      for (i = groups.length - 1; i >= 0; i--) {
+        if (collapse) {
+          this.collapseGroup(groups[i]);
+//        } else if (!groups[i].parent) {
+          } else {
+          this.uncollapseGroup(groups[i]);
+        }
+      }
+      this.update();
+    } else {
+      var i;
+      for (i = groups.length - 1; i >= 0; i--) {
```

Comment:
nit: for (var i =

Reply:
previous declaration of 'i' can be used here, so just removing the var i